stablehlo/transforms/CHLODecompositionPatterns.td

```diff
@@ -0,0 +1,383 @@
+// Copyright 2020 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// This is the legalization pattern definition file for CHLO to StableHLO.
```

Comment:
Rename file for consistency: `stablehlo/transforms/ChloDecompositionPatterns.td`

Reply:
Because CHLO decomposition includes canonicalize,I was afraid that I didn't send a notice to test it,so I added it,sorry.

Reply:
Ahhh. I see that now. OK we can proceed with this as is. Let's just do file renames and then we can add more fine-grained tests later. Ideally I'd like for CHLO decompositions to not depend on canonicalizations, but that will be a bit of a migration for the test files so we can do it separately